yaas/config.py

```diff
@@ -0,0 +1,11 @@
+# coding: utf-8
+
+# These default configs are overriden based on
+# command line arguments or environment variables
+
+server_url = None   # Must be overriden
+requests_kwargs = {
+    'auth': ('admin', 'admin'),
```

Comment:
Don't put defaults here, put them in generic form (`auth_user`, `auth_password`) in `config.py` and then build the request kwarg struct from that in `main()`.